### PR TITLE
Add favicon

### PR DESCRIPTION
**Pull request by kevinjihwanlee**

### CODE REVIEW

**Comment by kevinjihwanlee** [comment]
[Add favicon](https://trello.com/c/8oozgjEu/35-add-favicon)

**Comment by kevinjihwanlee** [comment]
screenshot 2018-11-11 12 59 21

File: dist/index.html [hunk 1]
```diff
@@ -21,6 +28,6 @@
       You need to enable JavaScript to run this app.
     </noscript>
 
-    <script src="./bundle.js"></script>
+    <script src="../dist/bundle.js"></script>
```
Comment (Joonpark13) [inline dist/index.html]: Don't think this change is necessary here since the file itself is in the `dist` folder.

File: webpack.dev.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ module.exports = env => merge(common(env), {
   devServer: {
     contentBase: path.join(__dirname, 'dist/'),
     port: 3000,
-    publicPath: 'http://localhost:3000/',
+    publicPath: 'http://localhost:3000/dist/',
```
Comment (Joonpark13) [inline webpack.dev.js]: Not sure why you changed this?

**Review by Joonpark13 — CHANGES_REQUESTED** [review]

File: webpack.dev.js (hunk 2 above)
Comment (kevinjihwanlee) [reply webpack.dev.js]: good point, forgot to revert that after we talked about why the project is structured the way it is!

File: dist/index.html (hunk 1 above)
Comment (kevinjihwanlee) [reply dist/index.html]: makes sense

**Review by Joonpark13 — APPROVED** [review]

**Pull request merged**